src/routes/settings.tsx

```
@@ -79,108 +79,99 @@ function SettingsPage() {
-                    />
-
-                    <div className="mt-2 flex w-full justify-between px-2 text-xs opacity-50">
-                        <span>Slow</span>
-                        <span>Default</span>
-                        <span>Fast</span>
+                {/* Server settings: saved to server-config.json on Save */}
+                <div className="space-y-6">
+                    <div className="form-control w-full">
+                        <label className="label">
+                            <span className="label-text">Port</span>
+                        </label>
```

Biome (2.3.14)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)